python/GangaLHCb/Lib/Server/DiracLHCbCommands.py

```diff
@@ -1,48 +1,29 @@
-import os
-import sys
-import time
-import datetime
-import glob
-import pickle
-import inspect
-## NB parseCommandLine first then import Dirac!!
-from DIRAC.Core.Base.Script import parseCommandLine
-parseCommandLine()
-from LHCbDIRAC.Interfaces.API.DiracLHCb import DiracLHCb
-from DIRAC.Interfaces.API.DiracAdmin import DiracAdmin
-
-diraclhcb = DiracLHCb()
-# Write to output pipe
-#/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
-
-#def output(object):
-#    print(pickle.dumps(object), file=sys.stdout)
 
 # DiracLHCb commands
 #/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
 
-#def getRootVersions(): output( diraclhcb.getRootVersions() )
+#def getRootVersions(): output( dirac.getRootVersions() )
 
-#def getSoftwareVersions(): output( diraclhcb.getSoftwareVersions() )
+#def getSoftwareVersions(): output( dirac.getSoftwareVersions() )
 
 
-def bkQueryDict(dict): output(diraclhcb.bkQuery(dict))
+def bkQueryDict(dict): output(dirac.bkQuery(dict))
```

Comment:
This file is embedded in another after the GangaDirac commands. This gets rid of a couple of hundred lines of repeated code between GangaDirac and GangaLHCb when doing Dirac work.